Source/Scene/GltfBufferViewLoader.js

```diff
@@ -229,6 +203,46 @@ function getBufferLoader(bufferViewLoader) {
   });
 }
 
+/**
+ * Processes the resources. For a BufferView that does not have the EXT_meshopt_compression extension, there
+ * is no processing that needs to happen, so this function returns immediately.
+ *
+ * @param {FrameState} frameState The frame state.
+ */
+GltfBufferViewLoader.prototype.process = function (frameState) {
+  //>>includeStart('debug', pragmas.debug);
+  Check.typeOf.object("frameState", frameState);
+  //>>includeEnd('debug');
+
+  if (!this._hasMeshopt) {
+    return;
+  }
+
+  if (!defined(this._typedArray)) {
+    return;
+  }
+
+  if (this._state !== ResourceLoaderState.PROCESSING) {
+    return;
+  }
+
+  var count = this._meshoptCount;
+  var byteStride = this._meshoptByteStride;
+  var result = new Uint8Array(count * byteStride);
+  MeshoptDecoder.MeshoptDecoder.decodeGltfBuffer(
+    result,
+    count,
+    byteStride,
+    this._typedArray,
+    this._meshoptMode,
+    this._meshoptFilter
+  );
+
+  this._typedArray = result;
+  this._state = ResourceLoaderState.READY;
+  this._promise.resolve(this);
```

Comment:
Changed so the promise returns the loader. It used to be `this._promise.resolve();`